window_management/window_tweak.talon

```diff
@@ -0,0 +1,121 @@
+
+# show size and position information about the current window
+win show$:
```

Comment:
Perhaps a different command name for this? `win show` sounds like it will show a window that's hidden, as opposed to show positioning info.

Reply:
Changed to 'window traits show' and 'window traits hide' - used 'traits' to keep the number of syllables to a minimum.